server/server.js

```diff
@@ -4,7 +4,7 @@ const db = require('../database/database');
 
 // Set up Express server and JSON parsing of API requests
 const app = express();
-app.use(express.static('client/dist'));
+app.use('/:id', express.static('client/dist'));
```

Comment:
That is the route that I show the deals page on. Any only urls that have an id attached to the end (like localhost:3002/1 or localhost:3002/2) will display the deals page. This is useful because I can pull the deal id from that url and use it to retrieve the corresponding deal information from the server using my API endpoints